src/view/screens/DiscoverFeeds.tsx

```diff
@@ -22,6 +23,33 @@ export const DiscoverFeedsScreen = withAuthRequired(
     const pal = usePalette('default')
     const feeds = React.useMemo(() => new FeedsDiscoveryModel(store), [store])
 
+    // search stuff
+    const [isInputFocused, setIsInputFocused] = React.useState<boolean>(false)
+    const [query, setQuery] = React.useState<string>('')
+    const onChangeQuery = React.useCallback(
+      (text: string) => {
+        setQuery(text)
+        if (text.length > 1) {
+          feeds.search(text)
```

Comment:
We need to debounce this query -- it's currently running on every change and the results kind of oddly update as you go, often with some lag.

Reply:
Fixed.